.flowconfig

```diff
@@ -0,0 +1,7 @@
+[ignore]
+
+[include]
+
+[libs]
+
+[options]
```

Comment:
[options]
sharedmemory.hash_table_pow=21

this will make webstorm works well with flow